sssoja/tv-bland
# remove unused helper file and folder

### What changes have you made?
- Renamed the subdirectory `styled` to `styles` for clarity
- Updated the pathnames of all `Grid`, `Flex` and `Padding` imports

### Which issue(s) does this PR fix?
Fixes #12 

### Screenshots (if there are design changes)
No design changes

### How to test

Check for a subdirectory in the components folder called `styles` with the following UI components: 
- `Grid`
- `Flex`
- `Padding`

When you open `localhost:3000` the updated pathnames should not throw any errors
